src/Raven.Studio/wwwroot/App/views/database/indexes/editIndex.html

```diff
@@ -216,6 +222,105 @@ <h2 id="reduce-title" class="margin-bottom margin-bottom-xs on-base-background">
                         </div>
                     </div>
                 </div>
+                <div role="tabpanel" class="tab-pane fade" id="additionalAssemblies">
+                    <div class="flex-form">
+                        <div class="row margin-left margin-bottom margin-bottom-lg">
+                            <button type="button" data-bind="click: addAssembly" class="btn btn-info margin-left"><i class="icon-plus"></i> <span>Add Assembly</span></button>
+                            <a href="#" data-bind="click: additionalSourceSyntaxHelp" class="margin-left margin-left-sm"><small>Syntax <i class="icon-help"></i></small></a>
+                        </div>
+                        <div data-bind="foreach: editedIndex().additionalAssemblies">
+                            <hr>
+                            <div class="row">
+                                <div class="col-lg-3">
+                                    <div class="form-group">
+                                        <label class="control-label">Assembly Source</label>
+                                        <div class="btn-group">
+                                            <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" title="Select assembly source type">
+                                                <span data-bind="text: assemblySource"></span>&nbsp;&nbsp;<span class="caret"></span>
+                                            </button>
+                                            <ul class="dropdown-menu" role="menu">
+                                                <li data-bind="click: _.partial(setAssemblySourceType, 'Server Runtime')"><a href="#">Server Runtime</a></li>
+                                                <li data-bind="click: _.partial(setAssemblySourceType, 'Path')"><a href="#">Path</a></li>
+                                                <li data-bind="click: _.partial(setAssemblySourceType, 'NuGet')"><a href="#">NuGet</a></li>
+                                            </ul>
+                                        </div>
+                                    </div>
+                                </div>
+                                <div class="col-lg-7">
+                                    <div class="form-group" data-bind="visible: assemblySource() === 'Server Runtime'">
+                                        <label class="control-label">Assembly Name <i class="required"></i></label>
+                                        <div class="flex-grow" data-bind="validationElement: assemblyName">
+                                            <input type="text" autocomplete="off" class="form-control" placeholder="Enter assembly name" data-bind="textInput: assemblyName" />
```

Comment:
can we put autocompete on form elemnt instead of putting this on every input?

Reply:
done